SegmentEditorSurfaceCut/SegmentEditorSurfaceCutLib/SegmentEditorEffect.py

```diff
@@ -377,7 +405,8 @@ def onSegmentEditorNodeModified(self, observer, eventid):
   def updateModelFromSegmentMarkupNode(self):
     if not self.segmentMarkupNode or not self.segmentModel:
       return
-    self.logic.updateModelFromMarkup(self.segmentMarkupNode, self.segmentModel)
+    smoothModelFlag = self.smoothModelCheckbox.isChecked()

```

Comment:
Since this method is kind of "logic" (not GUI), it is nicer if we don't refer to GUI widgets but get the smoothModel flag from the effect parameter instead.